.eslintrc.js

```diff
@@ -6,6 +6,9 @@ module.exports = mergeDeepLeft(
   {
     rules: {
       "@bigbinary/neeto/no-missing-localization": "off",
+      "@bigbinary/neeto/webpack-aliases-and-jsconfig-paths-should-be-in-sync":
+        "off",
+      "@bigbinary/neeto/ensure-compact-objects": "off",
```

Comment:
Why is this rule turned off?

Reply:
Yesterday, both the rules `@bigbinary/neeto/webpack-aliases-and-jsconfig-paths-should-be-in-sync` and `@bigbinary/neeto/ensure-compact-objects` were giving me errors similar to the `@bigbinary/neeto/no-missing-localization` rule when I attempted to commit changes.

However, upon rechecking after enabling them, it seems to be fixed. I'm not sure if this was a version issue or something else.